mteb/descriptive_stats/Classification/DutchGovernmentBiasClassification.json

```diff
@@ -0,0 +1,54 @@
+{
+    "test": {
+        "num_samples": 782,
+        "number_texts_intersect_with_train": 100,
+        "text_statistics": {
+            "total_text_length": 178084,
+            "min_text_length": 32,
+            "average_text_length": 227.72890025575447,
+            "max_text_length": 2746,
+            "unique_texts": 752
```

Comment:
I see that a few datasets have duplicate samples. Can you deduplicate these

Reply:
Hmm, yeah, but I would rather have that than a bad task. Alternatively you can filter smaller tasks in the dataset transform (with a comment stating why)